plugins/com.google.cloud.tools.eclipse.bugreport/src/com/google/cloud/tools/eclipse/bugreport/ui/BugReportCommandHandler.java

```diff
@@ -57,10 +60,20 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
   @VisibleForTesting
   static String formatReportUrl() {
     String body = MessageFormat.format(BODY_TEMPLATE, CloudToolsInfo.getToolsVersion(),
-        System.getProperty("os.name"), System.getProperty("os.version"),
+        getCloudSdkVersion(), System.getProperty("os.name"), System.getProperty("os.version"),
         System.getProperty("java.version"));
 
     Escaper escaper = UrlEscapers.urlFormParameterEscaper();
     return BUG_REPORT_URL + "?body=" + escaper.escape(body);
   }
+
+  private static Object getCloudSdkVersion() {
+    try {
+      CloudSdk sdk = new CloudSdk.Builder().build();
+      return sdk.getVersion();
+    } catch (AppEngineException ex) {
+      // report the exception toString() as it can be meaningful
```

Comment:
`MessageFormat.format()` already applies `toString()` to the objects presented.  A cooked example, where I threw `CloudSdkNotFoundException`:
```
- Cloud Tools for Eclipse version: 0.0.0
- Google Cloud SDK version: com.google.cloud.tools.appengine.cloudsdk.CloudSdkNotFoundException: test
- OS: Mac OS X 10.12.5
- Java version: 1.8.0_131
```

It means I can avoid importing additional packages.